symphony/bdk/core/config/model/bdk_server_config.py

```diff
@@ -0,0 +1,80 @@
+class BdkServerConfig:
+    """Base class for server and client configurations
+    """
+    _DEFAULT_SCHEME: str = "https"
+    _DEFAULT_HTTPS_PORT: int = 443
+
+    def __init__(self, scheme=_DEFAULT_SCHEME, port=_DEFAULT_HTTPS_PORT, context="",  host=None, **kwargs):
+        self._scheme = scheme
+        self._port = port
+        self.context = context
+        self._host = host
+
+    def get_base_path(self) -> str:
+        """Constructs the base path of the current config
+
+        :return: scheme://host:port + formated_context
+        :rtype: str
+        """
+        return self.scheme + "://" + self.host + self.get_port_as_string() + self.get_formated_context()
+
+    def get_formated_context(self) -> str:
+        """Formats the context field
+
+        :return: "/" + context if the context is not empty
+        :rtype str
+        """
+        local_context = self.context
+        if local_context == None:
+            return ""
+        if local_context and local_context[0] != "/":
```

Comment:
is it safe to access [0] without checks?

Reply:
I don't think it will work if the context is an int ... 
I will add a condition to check if local_context is a string.